contracts/ts/__tests__/utils.ts

```diff
@@ -1,4 +1,14 @@
+import { SNARK_FIELD_SIZE } from "maci-crypto"
+
 export async function timeTravel(provider, seconds) {
     await provider.send('evm_increaseTime', [Number(seconds)])
     await provider.send('evm_mine', [])
 }
+
+export const validateSaltFormat = (salt: string): boolean => {
```

Comment:
Why are these methods necessary?

Reply:
- `timeTravel` is needed to move forward the state machine time for the local blockchain node. 
- `validateSaltFormat` is not needed anymore since I'm using a fixed valid value for the `salt`. Let me remove it!

Reply:
done here https://github.com/0x3327/maci/pull/33/commits/e9a120a4ba8be54520281719c0df28f9ff38b4a1